docs/gitflow.md

```diff
@@ -18,3 +18,16 @@ We use [Gitflow](https://docs.github.com/en/get-started/quickstart/github-flow)
 While a release is pending, we may merge anything into develop, **except for changes to package.json files**. If we
 change the package.json files on develop, the gitflow PR master -> develop will have merge conflicts, because during the
 release the package.json files are updated on master.
+
+## What to do if there is a merge conflict?
+
+Although gitflow should help us to avoid merge conflicts, as mentioned above in "Important Caveats" it can still happen
+that you get a merge conflict when trying to merge master into develop after a successful release.
+
+If this happen, you can resolve this as follows:
+
+- Close the automated PR that was created by the gitflow automationxw
+- Create a new branch on top of `master` (e.g. `manual-develop-sync`)
+- Merge `develop` into this branch, with a merge commit
```

Comment:
Should we mention fixing the merge conflict? 😅

Reply:
good point, I guess :D